### DISCUSSION TITLE
我加入了select2这个插件，让select支持多选，请问我应该如何再lay-submit的时候，触发这个select2的单验证啊？

### DISCUSSION THREAD

**Discussion by coolqingcheng** [layui/layui#1777] [问答]
或者说有其他的方法让layui的表单，支持其他类input的插件的验证

**Comment by LSL1618** [comment]
是这个 [select2](https://select2.org/) 吧，它是第三方的小组件，要触发验证的话应该要使用它自带的属性或者方法，看看 [select2属性文档](https://select2.org/configuration/options-api)
